src/template/page-home.php

```diff
@@ -111,6 +111,9 @@
                   <svg class="c-icon -small"><use xlink:href="#icon-download"></use></svg>
                 </button>
                 <?php renderDownloadOptions() ?>
+                <button class="btn -primary">
```

Comment:
Remove the link from the button. It doesn't work on Firefox. You can attach the button's classes to the link directly.